### PR TITLE
LPS-204334 Fix API Builder tests

### PR DESCRIPTION
**Pull request by 4lejandrito**
https://liferay.atlassian.net/browse/LPS-204334

### CODE REVIEW

**Comment by 4lejandrito** [comment]
ci:forward

**Comment by magjed4289** [comment]
checking the relevant results 👁️

**Comment by magjed4289** [comment]
ci:help

**Comment by magjed4289** [comment]
ci:test:headless-builder

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java [hunk 1]
```diff
@@ -580,44 +581,55 @@ private void _initSystemObjectDefinition(
...
 		return TransformUtil.transform(
-			Arrays.asList(
-				new ObjectAssetCategoryExceptionMapper(),
-				new ObjectEntryCountExceptionMapper(),
-				new ObjectEntryManagerHttpExceptionMapper(),
-				new ObjectEntryStatusExceptionMapper(_language),
-				new ObjectEntryValuesExceptionMapper(_language),
-				new ObjectRelationshipDeletionTypeExceptionMapper(_language),
-				new ObjectValidationRuleEngineExceptionMapper(
+			Arrays.<Supplier<ExceptionMapper<?>>>asList(
```
Comment (magjed4289) [inline modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java:591]: in a spare time I'd like to ask you for the details of the approach, as this function is little hard to understand for me. But all the tests of API Builder are passing now, so I'll forward ✅

**Review by magjed4289 — APPROVED** [review]

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java (hunk 1 above)
Comment (4lejandrito) [reply modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java:591]: hey @magjed4289 this is a change to a private method... I think that semantic change is not related.

**Comment by 4lejandrito** [comment]
ci:forward:force

**Pull request closed without merging**